src/frontend/src/session.rs

```diff
@@ -178,6 +178,9 @@ pub(crate) struct FrontendEnv {
 
     /// Memory context used for batch executors in frontend.
     mem_context: MemoryContext,
+
+    /// address of the serverless backfill controller.
+    sbc_addr: String,
```

Comment:
The abbreviation of `sbc` is a bit confusing and hard to maintain; at first glance, I didn't know what it meant. Can we use the full name instead?